CHANGELOG.md

```diff
@@ -181,14 +198,72 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
 
 #### Core
 
-* We deprecated seqan3::range_compatible_concept and it will be removed in 3.1.0
+* Deprecated seqan3::range_compatible_concept and it will be removed in 3.1.0
   ([\#2265](https://github.com/seqan/seqan3/pull/2265)).
 
+Header Changes:
+
+```cpp
+#include <seqan3/core/algorithm/bound.hpp> [Functionality included in alignment/configuration/align_config_band.hpp]
+#include <seqan3/core/{algorithm => configuration}/configuration.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/concept.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_debug_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_parallel_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_utility.hpp>
+#include <seqan3/core/{algorithm => configuration}/pipeable_config_element.hpp>
+
+#include <seqan3/{core/bit_manipulation => std/bit}.hpp>
+#include <seqan3/{core/bit_manipulation => utility/detail/bits_of}.hpp>
+#include <seqan3/{core/bit_manipulation => utility/detail/to_little_endian}.hpp>
+
+#include <seqan3/{core => utility}/char_operations/predicate.hpp>
+#include <seqan3/{core => utility}/char_operations/predicate_detail.hpp>
+#include <seqan3/{core => utility}/char_operations/pretty_print.hpp>
+#include <seqan3/{core => utility}/char_operations/transform.hpp>
+
+#include <seqan3/{core => utility/tuple}/common_tuple.hpp>
+
+#include <seqan3/{core/concept/core_language => utility/detail/exposition_only_concept}.hpp>
+#include <seqan3/{core/concept/tuple => utility/tuple/concept}.hpp>
+
+#include <seqan3/{core => utility}/math.hpp>
+
+#include <seqan3/{core => utility/tuple}/pod_tuple.hpp>
+
+#include <seqan3/{core => utility}/simd/concept.hpp>
+#include <seqan3/{core => utility}/simd/{ => detail/}debug_stream_simd.hpp>
+#include <seqan3/{core => utility}/simd/{simd_algorithm => algorithm}.hpp>
+#include <seqan3/{core => utility}/simd/simd.hpp>
+#include <seqan3/{core => utility}/simd/simd_traits.hpp>
+#include <seqan3/{core => utility}/simd/{view_iota_simd => views/iota_simd}.hpp>
+#include <seqan3/{core => utility}/simd/{view_to_simd => views/to_simd}.hpp>
```

Comment:
```suggestion
#include <seqan3/{core => utility}/simd/concept.hpp>
#include <seqan3/{core => utility}/simd/{simd_algorithm => algorithm}.hpp>
#include <seqan3/{core => utility}/simd/simd.hpp>
#include <seqan3/{core => utility}/simd/simd_traits.hpp>
#include <seqan3/{core => utility}/simd/{view_iota_simd => views/iota_simd}.hpp>
#include <seqan3/{core => utility}/simd/{view_to_simd => views/to_simd}.hpp>
```

#include <seqan3/{core => utility}/simd/{ => detail/}debug_stream_simd.hpp>
is detail.